magenta-lib/src/main/scala/magenta/deployment_type/CloudFormation.scala

```diff
@@ -157,21 +158,27 @@ object CloudFormation extends DeploymentType with CloudFormationDeploymentTypePa
       )
     )
 
+    val policy = target.parameters.updateStrategy match {
```

Comment:
The naming of this val here doesn't make it very clear what the policy is for.

Reply:
renamed to updatePolicy